docs/guide/content_rendering.md

```diff
@@ -1047,6 +1045,125 @@ These parameters can then be used in templates, for example:
 {% endblock %}
 ```
 
+### Adding a listener
+
+One way to keep control of what is passed to the view is to use your own listener. It can be easily used to apply some custom logic to all views, but it is possible to limit the scope in any way required.
+To use the listener you need to add it to the configuration, for example:
+
+``` yaml
+services:
+    my.custom.view_listener:
+        class: AcmeTestBundle\Listener\MyViewListener
+        tags:
+            - { name: kernel.event_subscriber }
+```
+
+With this configuration, the following listener will add some additional parameter to the view. It needs to listen to `MVCEvents::PRE_CONTENT_VIEW`:
+
+``` php
+// Listener
+<?php
+
+namespace Acme\TestBundle\Listener;
+
+use eZ\Publish\Core\MVC\Symfony\Event\PreContentViewEvent;
+use eZ\Publish\Core\MVC\Symfony\MVCEvents;
+use Symfony\Component\EventDispatcher\EventSubscriberInterface;
+
+class MyViewListener implements EventSubscriberInterface
+{
+    public static function getSubscribedEvents()
+    {
+        return [MVCEvents::PRE_CONTENT_VIEW => 'doSomethingCustom'];
+    }
+
+    public function doSomethingCustom(PreContentViewEvent $event)
+    {
+        $view = $event->getContentView();
+
+        // Add custom parameters to existing ones.
+        $view->addParameters(['anotherCustomVariable' => "Another message!"]);
+
+        // If you wish, you can also easily access Location and Content objects
+        $location = $view->getLocation();
+        $content = $view->getContent();
+    }
+}
+```
+
+Similarly, you can use a listener to modify the response generated after the view is rendered.
```

Comment:
Why do we need this example? It overlaps with what we do in the product already and will easily lead to confusion.
ref: https://github.com/ezsystems/ezplatform-http-cache/tree/master/src/EventSubscriber

Reply:
I have added this for the sake of completion - it describes how to modify the Response after the view is rendered. I have described in my previous comment why I think it is a good idea, at least as an option.
It doesn't have to concern cache (and you are right, I guess it shouldn't). I was just following the previous examples. I will change it to show how (for example) to add a custom Response header.

Reply:
I changed it to also add the custom Response header instead of dealing with cache. Also simplified the example a bit.

Reply:
AS I think @bdunogier mentioned you can drop this, in it's current form it is not clear when the user would need to do this.

So maybe rather change it, so it so the folioing in list at the top:
`- [Add a Symfony Response listener](#adding-a-listener) to add custom logic to all responses`

And section here can be cut down to just point to Symfony documentation for creating response listeners.

Reply:
Done.